docs/en/latest/discovery.md

```diff
@@ -182,6 +182,8 @@ discovery:
 
 ## Upstream setting
 
+### routes
```

Comment:
We should use L4 and L7 instead of routes, as the configuration actually takes effect in the upstream

Reply:
sorry, i will change it.